Gruntfile.js

```diff
@@ -447,43 +462,69 @@ module.exports = function( grunt ) {
 					packageManager: "npm"
 				}
 			}
+		},
+		"start-selenium-server": {
+			dev: {
+				options: {
+					downloadUrl: "https://selenium-release.storage.googleapis.com/2.45/" +
+						"selenium-server-standalone-2.45.0.jar",
+					downloadLocation: "node_modules/grunt-selenium-server/",
+					serverOptions: serverOptions,
+					systemProperties: {}
+				}
+			}
+		},
+		"stop-selenium-server": {
+			dev: {}
```

Comment:
Is this (selenium + chromedriver) a requirement for having tests to run on browserstack? Or is it a requirement for having tests to run on Chrome locally?

Isn't `'browserstack.selenium_version': '2.45.0'` configuration sufficient? ([more info](https://github.com/theintern/intern/blob/master/tests/example.intern.js#L12-L14)).